.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+<!-- Please provide all the information below.  -->
+
```

Comment:
I stole this from the functions host repo.

May want a context section here? I prefer context at the top rather than under "Additional Information" below.

Reply:
Restructured this pull request template a bit, including some instructions for when you do squash merge it. Let me know what you all think.

Reply:
Makes sense to me!